app/views/groups/samples/_table.html.erb

```diff
@@ -90,16 +98,40 @@
         class="bg-white border-b dark:bg-slate-800 dark:border-slate-700"
       >
         <td class="px-6 py-3 whitespace-nowrap left-0 sticky bg-white dark:bg-slate-800">
-          <%= link_to sample.name,
-                      namespace_project_sample_path(
+          <%= link_to namespace_project_sample_path(
+                        id: sample.id,
+                        namespace_id: sample.project.namespace.parent.full_path,
+                        project_id: sample.project.path
+                      ),
+                      data: {
+                        turbo: false
+                      },
+                      class: "text-slate-900 dark:text-slate-100 font-semibold hover:underline" do %>
+            <span class="font-semibold">
+              <%= highlight(
```

Comment:
**praise:** I love this addition (I actually have a tab open with this and was planning on updating the tables with it), thank  you.